src/main.py

```diff
@@ -107,6 +107,15 @@ def list_group_categories(db: DbDependency, group_id: int):
 
     return categories
 
+@app.put("/category", status_code=HTTPStatus.OK)
```

Comment:
Estamos pasando la primary key como _path parameter_ en estos casos.

```suggestion
@app.put("/category/{category_name}")
```